### DISCUSSION TITLE
HELP WANTED: This project is seeking a new home

### DISCUSSION THREAD

**Discussion by ZacSweers** [slackhq/keeper#144] [General]
We no longer use Keeper at Slack. It works fine, we just no longer run minified tests (for unrelated, nontechnical reasons). As such, we're not actively using or maintaining this. If your company uses this and is interested in taking over development in a fork, please comment here and we'll reach out if it seems like a good fit. Unfortunately the powers that be don't allow us to transfer the repo, but we'd be happy to update the README to point at the new home once one is found.

**Comment by joshfriend** [comment]
I'm interested in picking this up. Square is still using it and we're still maintaining minified UI tests.

**Comment by ryanholden8** [comment]
Hey guys, just checking if there has been any update on this? Our team would like to keep using Keeper but we'd like to know if it's a stable project for us to continue forward?

**Reply by ZacSweers** [reply]
It's not abandoned, we just want to rehome it long term

**Reply by iliaskomp** [reply]
Hi, is there any update on someone taking over keeper?
